server/src/config/client/src/pages/Contact.scss

```diff
@@ -9,26 +9,55 @@
       align-items: flex-start;
 
       .form-inputs {
-        margin-bottom: 8rem;
         width: 100%;
         font-size: 1.2rem;
+        border: none;
+        margin-bottom: 8rem;
 
         input[type=text], input[type=email] {
         background-color: $background;
         color: $secondaryText;
-        }
```

Comment:
This could be within the base css of .form-input since there aren't other forms that need a different background color and text color.